tests/app/authentication/test_authentication.py

```diff
@@ -28,6 +28,12 @@
 from tests.conftest import set_config, set_config_values
 
 
+def create_custom_jwt_token(headers=None, payload=None, key=None):
+    # code copied from notifications_python_client.authentication.py::create_jwt_token
+    headers = headers or {"typ": 'JWT', "alg": 'HS256'}
+    return jwt.encode(payload=payload, key=key or str(uuid.uuid4()), headers=headers)
```

Comment:
I did actually put the `key` default in the parameters, but then `flake8` complained about it for the reason you gave. `headers` is more superficial: I just didn't want to clutter the parameter list with a big literal.

`flake8` makes a good point about `key`, and since having `dict` defaults don't scale (as the literal gets larger), I think this is probably the most consistent way to write such a function.

I'll resolve this one for now, and @leohemsted can say if he can think of another way of doing it.